src/pages/index.js

```diff
@@ -1,36 +1,40 @@
 import React from 'react';
 import GatsbyLink from 'gatsby-link';
-import Helmet from 'react-helmet';
+import { graphql } from 'gatsby';
```

Comment:
Avoids a console warning that `import { graphql } from 'gatsby'` should now be used.

This seems a lot cleaner _and_ it means this won't have to be mocked out in tests and has some other nice benefits. No "magic" globals!